README.md

```diff
@@ -6,147 +6,127 @@
 
 ## Installation
 
+You can add Swift SDK to your application in two ways: via **SPM** or **CocoaPods**.
+
 ### SPM
 
-1. Open Xcode & Add the SDK
-2. Open your Xcode project.
-3. Go to File → Add Packages.
-4. In the search bar, enter the repository URL of the SDK:
+In the Xcode, when your project is opened, select **File → Add Package Dependencies...**
+
+In the the opened window enter URL of the Atlas SDK in the search bar:
 
 ```
 https://github.com/atlas-support-inc/mobile-sdk-ios
 ```
 
-5. Select the SDK package from the list.
-6. Click Add Package. Do not forget "Add to Target".
-7. Choose Dependency Rule (e.g., "Up to Next Major Version" or "Exact Version").
+Select the **AtlasSupportSDK** from the search results, configure **Dependency Rule** and **Add to Project** settings, and click **Add Package** button.
 
 ### CocoaPods
 
-Add the SDK to your `Podfile`:
+In your project add **AtlasSupportSDK** dependency to the **Podfile**:
 
-```ruby
+```swift
 platform :ios, '13.0'
 use_frameworks!
 
 target 'YourApp' do
-  pod 'AtlasSupportSDK', '~> 0.0.6'
+  pod 'AtlasSupportSDK', '~> 0.1.2'
 end
 ```
 
-Run the following command in your terminal:
+Pull CocoaPods dependencies:
 
-```bash
+```
 pod install
 ```
 
-Open the `.xcworkspace` file in Xcode.
-
 ---
 
-## Usage
-
-### Import the SDK
+## Setup
 
-In any file where you want to use the SDK, import it:
+Import the package into your code:
 
 ```swift
 import AtlasSupportSDK
 ```
 
-In this example, we import the SDK into HomeViewController and TabBarController because these are the two navigation points leading to the AtlasViewController.
+Connect the SDK to your account:
 
----
+```swift
+AtlasSDK.setAppId("YOUR_APP_ID")
+```
 
-### Initialize and Configure the SDK
+---
 
-Call the appropriate initialization and configuration methods erly. Depends on your app architecture.
+## Identify your users
 
-For example:
+Make the following call with user details wherever a user logs into your application and you get the user details for the first time.
 
 ```swift
-AtlasSDK.setAppId("YOUR_APP_ID")
+AtlasSDK.identify(userId: "...",
+    userHash: nil,
+    name: "...",
+    email: "...",
+    phoneNumber: "...")
 ```
 
-You can find `YOUR_APP_ID` in the Atlas application: https://app.atlas.so/settings/company
+To make sure scammer can't spoof a user, you should pass a `userHash` into identify call (read more about it here).
 
----
+Unknown properties, like `name` or `phoneNumber` should be set to nil (not empty string) so they won't be overriden if they have previously been stored.
+
+When you want to update the user's details you can call `identify` method again.
+
+Additionally, there is a `logout` method available to clear the user's session when they log out of your application.
+
+```swift
+AtlasSDK.logout()
+```
 
-### Presenting the Atlas Chat ViewController
+## Show UI
```

Comment:
SwiftUI

Reply:
Oh, I understand. This title should clearly say "UIKit" or "ViewController", otherwise it's confusing.
Revert this change